### DISCUSSION TITLE
Search by file extension

### DISCUSSION THREAD

**Discussion by arnabc74** [jarun/nnn#1664] [Q&A]
This must be a silly question with a simple answer, which is somehow eluding me. I want to list all files with extension .rb in the current folder. So the natural instinct is to type /.rb . But unfortunately, I have many files with extension .rb~, .rb.sav1, etc. I want to exclude those. If I could use regular expression I could have used /.rb$ (or something like that). But nnn would not let me use regexp. 

Any idea?

**Comment by N-R-K** [comment]
`nnn` supports regex. Press `/` two times and you should be able to use regex.

Or you can use the `-g` cli flag. Open up the nnn manpage and search for "regex" there's good documentation on it. (Our wiki also contains a lot of in depth documentation).
